.github/ISSUE_TEMPLATE/website-technical-issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: 🏗️ Documentation Website Technical Issue
+about: Issues with the documentation website build, aggregation, or infrastructure (not content)
+title: '[WEBSITE] '
+labels: ['documentation-website']
+---
+
+## Documentation Website Technical Issue
+
+**⚠️ Note: This template is only for technical issues with the documentation website itself (build failures, broken links, aggregation problems, etc.)**
+
+**For documentation content issues, please use the appropriate source repository:**
+- **Infrahub core docs**: https://github.com/opsmill/infrahub/issues/new/choose
+- **Python SDK docs**: https://github.com/opsmill/infrahub-sdk-python/issues/new  
+- **Ansible collection docs**: https://github.com/opsmill/infrahub-ansible/issues/new
+
+---
+
+### What type of website issue is this?
+<!-- Please check one -->
+- [ ] Build failure or deployment issue
+- [ ] Broken links between pages
+- [ ] Documentation aggregation/sync problem
+- [ ] Website performance issue
+- [ ] Other technical website issue
+
+### Description
+<!-- Describe the technical issue with the website -->
+
+### Steps to reproduce
+<!-- If applicable, steps to reproduce the issue -->
+1. 
+2. 
+3. 
+
+### Expected behavior
```

Comment:
Let's keep it simple, and remove `Expected behavior` and `Actual behavior` for now